app/views/teams/_sub_teams.html.erb

```diff
@@ -19,9 +19,8 @@
     <% end %>
   </div>
 <% end %>
-
-<div id="team-list-desktop" class="flex flex-col gap-4 pt-6">
+<ul class="team-list flex flex-col gap-4 mt-6 overflow-y-auto max-h-[240px]">
```

Comment:
what is the use of team-list class ?, have you tested the code ? If you remove the "id" then adding the new item won't reflect in the list.

Reply:
It is a typo, even when I tested couldn't find any issue ,so missed it

Reply:
ID is replaced